app/scripts/modules/titus/src/instance/details/TitusInstanceDetails.tsx

```diff
@@ -0,0 +1,202 @@
+import { buildTaskActions, extractHealthMetrics } from './titusInstanceDetailsUtils';
+
+export interface ITitusInstanceDetailsProps extends IInstanceDetailsProps {
+  instance: { instanceId: string };
+}
+
+export interface ITitusInstanceDetailsContentProps {
+  account: IAccountDetails;
+  actions: Action[];
+  app: Application;
+  baseIpAddress: string;
+  environment: string;
+  healthMetrics: IAmazonHealth[];
+  instance: ITitusInstance;
+  instanceId: string;
+  instancePort: string;
+  loading: boolean;
+  moniker: IMoniker;
+  serverGroup: ITitusServerGroup;
+  region: string;
+}
+
+const TitusInstanceDetailsContent = ({
+  account,
+  actions,
+  app,
+  baseIpAddress,
+  environment,
+  healthMetrics,
+  instance,
+  instanceId,
+  instancePort,
+  loading,
+  moniker,
+  serverGroup,
+  region,
+}: ITitusInstanceDetailsContentProps) => {
+  const defaultSshLink = `ssh -t ${account?.bastionHost} 'titus-ssh -region ${region} ${instanceId}'`;
+  const regionDetails = (account.regions || []).find((r) => r.name === region);
+
+  return (
+    <div className="details-panel">
+      <div className="header">
+        <InstanceDetailsHeader
+          healthState={instance.healthState}
+          instanceId={instanceId}
+          loading={loading}
+          sshLink={defaultSshLink}
+          standalone={app.isStandalone}
+        />
+        <div className="actions">
+          <InstanceActions actions={actions} />
+          {Boolean(instance?.insightActions?.length) && (
+            <InstanceInsights analytics={true} insights={instance.insightActions} instance={instance} />
+          )}
+        </div>
+      </div>
+      <div className="content">
+        <TitusInstanceInformation instance={instance} titusUiEndpoint={regionDetails?.endpoint} />
+        <InstanceStatus
+          healthMetrics={healthMetrics}
+          healthState={instance.healthState}
+          metricTypes={['TargetGroup']}
+          privateIpAddress={instance.privateIpAddress}
+        />
+        <TitusInstanceDns
+          containerIp={instance.placement?.containerIp}
+          host={instance.placement?.host}
+          instancePort={instancePort}
+          ipv6Address={instance.ipv6Address}
+        />
+        <div className="collapsible-section">
+          <TitusSecurityGroupsDetailsSection app={app} serverGroup={serverGroup as ITitusServerGroupView} />
+        </div>
+        <CollapsibleSection heading="Console Output">
+          <ConsoleOutputLink instance={instance} />
+        </CollapsibleSection>
+        <InstanceLinks
+          address={baseIpAddress}
+          application={app}
+          instance={instance}
+          environment={environment}
+          moniker={moniker}
+        />
+      </div>
+    </div>
+  );
+};
+
+const OverridableTitusInstanceDetailsContent = overridableComponent<
+  ITitusInstanceDetailsContentProps & IOverridableProps,
+  typeof TitusInstanceDetailsContent
```

Comment:
`typeof TitusInstanceDetailsContent` oh interesting, I wonder if we can get `overridableComponent` to automatically infer this